core/block_dragger.js

```diff
@@ -227,3 +231,29 @@ Blockly.BlockDragger.prototype.updateCursorDuringBlockDrag_ = function() {
     }
   }
 };
+
+/**
+ * TODO (fenichel): Consider putting this in utils.
+ * Convert a coordinate object from pixels to workspace units, including a
+ * correction for mutator workspaces.
+ * This function does not consider differing origins.  It simply scales the
+ * input's x and y values.
+ * @param {!goog.math.Coordinate} pixelCoord A coordinate with x and y values
+ *     in css pixel units.
+ * @return {!goog.math.Coordinate} The input coordinate divided by the workspace
+ *     scale.
+ * @private
+ */
+Blockly.BlockDragger.prototype.pixelsToWorkspaceUnits_ = function(pixelCoord) {
+  var result = new goog.math.Coordinate(pixelCoord.x / this.workspace_.scale,
+      pixelCoord.y / this.workspace_.scale);
+  if (this.workspace_.isMutator) {
+    // If we're in a mutator, its scale is always 1, purely because of some
```

Comment:
We did it because the bubble canvas is inside a workspace's svg, not parallel to it, and is therefore impacted by its scale.  It's definitely not a specific goal of ours.  If you want I can make a cleanup issue around resolving that, and put a todo-link to it in this comment.